src/editors/containers/VideoEditor/components/VideoSettingsModal/index.jsx

```diff
@@ -32,19 +32,20 @@ export const VideoSettingsModal = ({
         textDecoration: 'none',
         marginLeft: '3px',
       }}
+      disabled
     >
       <Icon src={ArrowBackIos} style={{ height: '13px' }} />
       <FormattedMessage {...messages.replaceVideoButtonLabel} />
     </Button>
     <ErrorSummary />
-    <VideoPreviewWidget />
+    {/* <VideoPreviewWidget /> */}
```

Comment:
Instead of comment couldn't we remove it

Reply:
Removed.